client/src/Pages/Recipe/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useParams } from 'react-router'
+import { RecipeTab } from 'Components/RecipeTab'
+import { Recipe } from 'lib/types'
+import breadSandwich from 'lib/mock/bread_sandwich.jpg'
+
+interface ParamTypes {
+  id: string
+}
+
+const mockRecipe: Recipe = {
+  name: 'Bread Sandwich',
+  relatedCharacters: ['Broccoli Girl', 'Banana Man'],
+  ingredients: ['Bread', 'Bread', 'Bread'],
+  description: 'A simple lunchtime meal that will fill you up',
+  image: breadSandwich
+}
+
+const RecipeOverview: React.FC = () => {
+  const { id } = useParams<ParamTypes>()
+  console.log(id)
```

Comment:
Generally you don't want to be committing console.logs it adds noise for other devs since they likely won't want to see this in their console.

Reply:
That makes sense. However, the app won't compile unless I use id somewhere in the code (or comment the useParams line out, but then it won't compile with the ParamTypes definition and import line). Hopefully it's fine to leave it there until I write the actual logic to retrieve recipes?

Reply:
All g, removed 4 u